test/provision_test.py

```diff
@@ -1,5 +1,7 @@
 import json
+
 from test.test_util import TestUtils
+
 import pytest
 
 from autoprotocol.protocol import Protocol
```

Comment:
nit: sort these imports alphabetically as well?

Reply:
Yup, see my comment. In this case, since they separated into 3 groups:
1. system, python-level default libs -> 'test'
2. external python package -> 'pytest'
3. internal references -> autoprotocol